content/review/document-level-permissions.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Document level Permissions"
+metaTitle: "sensenet - Document level Permissions"
+metaDescription: "sensenet Document level Permissions"
+---
+
+sensenet offers you the ability of setting permissions even on document level in addition to the [role-based permission](/concepts/role-based-permissions) logic.
+# Advantage
+Using content (or document) level permissions, you can set who has access to an individual content and allow/restrict actions one by one.
```

Comment:
Maybe we could go into this a bit deeper (inheritance, breaking the inheritance with document level permissions and use case(s) where it might be a good solution)

Reply:
Since sensenet has a tree based hierarchical content structure it ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. This means, if you set a permission on a document library it will be automatically applied to subfolders and contents. You are able to set a permission for an individual document or on whole libraries and workspaces. 
This can be a good solution, for example in case a certain user has the access to a workspace, but not to a certain folder in that workspace.

Reply:
admin ui-on és api-in keresztül is lehet

Reply:
to edit the permission settings for the them -> to edit the permission settings for them

Documentlevel -> Document level